### ISSUE TITLE
Expert to fetch HTTP resources from URL

### ISSUE THREAD

**Issue by wagner-certat** [certtools/intelmq#1558]
An expert to fetch the HTTP (or more protocols?) header/content from a URL, e.g. `source.url`. Could be processed for filtering later.

**Comment by gethvi** [comment]
I was thinking about something similar. Suggested features:
* downloading a file and calculating it's hash (for malware URLs without hash)
* taking a screenshot (for phishing URLs)

**Comment by wagner-certat** [comment]
> downloading a file and calculating it's hash (for malware URLs without hash)

Good idea, could be the done by same bot. (maybe "Resource fetcher" or so?)

> taking a screenshot (for phishing URLs)

-> #1048
